### ISSUE TITLE
Godot 4: Vulkan doesn't see environment

### ISSUE THREAD

**Issue by Pete-The-Penguin** [godotengine/godot#46478]
**Godot version:**
godot-4-dev

**OS/device including version:**
Kubuntu, Vulkan. Graphics Card: RX 580X

**Issue description:**
When running the project, the window only displays objects, lights, etc. but not the environment.

**Steps to reproduce:**
1. Make a new project
2. Run said project

**Minimal reproduction project:**
[GodotTest.zip](https://github.com/godotengine/godot/files/6055038/GodotTest.zip)

If you have any questions, let me know and i will anwser them.

**Comment by Calinou** [comment]
This is because the default environment is now editor-only.

**Comment by Pete-The-Penguin** [comment]
oh... is there a way to make an environment that the viewport can see?

**Comment by Calinou** [comment]
Yes, add a WorldEnvironment node to your scene and add a new Environment resource to it.

**Comment by Pete-The-Penguin** [comment]
ok, i'll try that and tell you how it goes.

**Comment by Pete-The-Penguin** [comment]
okay, while trying to get the viewport to display something other then black (which it did) i then tried setting the background to canvas which made the editor crashed. still couldn't make it display the sky, only other colors.

**Comment by Pete-The-Penguin** [comment]
i should also mention that the viewport only see's objects when a light is on it.

**Comment by Pete-The-Penguin** [comment]
finally got the viewport to see the sky! it was a little tricky though, maybe add something in the docs and or in a hint of some kind?

**Comment by clayjohn** [comment]
I can't reproduce any of the above stated issues. 

Windows 10, RX 5600 xt, built from latest at the time of writing 165d77a

**Comment by Pete-The-Penguin** [comment]
strange, cause i tried twice last time and got the same results, did you use the project i linked? or did you make your own project for testing?

**Comment by clayjohn** [comment]
I made a new project. Your instructions say to use an empty project. 

What version of the engine are you using?

**Comment by Pete-The-Penguin** [comment]
it says it in the title

**Comment by clayjohn** [comment]
Godot 4 changes multiple times per day. Because it hasn't released yet. Please link the commit that you built from.

**Comment by Pete-The-Penguin** [comment]
i don't know the commit that i built from, i'm gonna update my dev build of godot 4 tomorrow and see if the new commit fixed the problem for me.

**Comment by Pete-The-Penguin** [comment]
i built the latest commit, and i have the same problem, but adding a world environment fixes it. i think its just a documentation issue and not a rendering one.

**Comment by clayjohn** [comment]
Tagged as junior job. 

Once we start on the big documentation push, we should document the fact that default_environment.tres is gone in the Environment documentation and probably the 3D rendering document as well.

**Comment by Pete-The-Penguin** [comment]
sounds good

**Comment by tnt-ndriqimrr** [comment]
I feel like they should have announced in BIG letters wherever they made this change. Converting a project from Godot 3.x to 4.x and a black screen coming up, oh man did my heart just drop!

**Comment by Calinou** [comment]
> I feel like they should have announced in BIG letters wherever they made this change. Converting a project from Godot 3.x to 4.x and a black screen coming up, oh man did my heart just drop!

The default environment project setting still exists, but it's set to an empty value by default. I'm surprised the project converter isn't bringing its value along with it though.
